README.md

```diff
@@ -4,66 +4,52 @@ Tailwind plugin utilizing Daikin-specific styles
 
 ## Installation
 
-Install the plugin from npm:
+Install the plugin and tokens from npm:
 
-```javascript
+```sh
 npm install -D @daikin-oss/tailwind
+npm install @daikin-oss/dds-tokens
 ```
 
-Then add to your tailwind configuration file:
+Use the plugin in your tailwind config:
 
-```javascript
-const daikinPlugin = require('@daikin-oss/tailwind');
+```js
+const daikinPlugin = require("@daikin-oss/tailwind");
 
 module.exports = {
-    content: ['./src/**/*.js'],
-    theme: {},
-    variants: {
-        extend: {}
-    },
-    plugins: [daikinPlugin(), ...otherPlugins]
+  content: ["./src/**/*.js"],
+  theme: {},
+  variants: {
+    extend: {},
+  },
+  plugins: [daikinPlugin(), ...otherPlugins],
 };
 ```
 
+Import CSS variables:
+
+```js
+import "@daikin-oss/dds-tokens/css/daikin/Light/variables.css";
+```
+
+See [dds-tokens repository](https://github.com/dsv-rp/dds-tokens?tab=readme-ov-file#readme) for more information.
+
 ## Basic Usage
 
 ### Colors
 
 You can use extended colors anywhere you usually would.
 
-```javascript
-<button class="text-daikinBlue-500 border-daikinBlue-500 hover:bg-daikinBlue-100">
-    My Button
+```html
+<button
+  class="bg-ddt-color-common-brand-default hover:bg-ddt-color-common-brand-hover"
```

Comment:
We need to have documentation on zeroheight for the tailwind plugin. 
This can be a separate ticket, but should be done as part of the "tailwind plugin release"

Reply:
Created a ticket.
https://daikinsvoil.atlassian.net/browse/DDS-1651